### ISSUE TITLE
Previous thumbnail overlaps accessibility

### ISSUE THREAD

**Issue by jaggzh** [almalence/OpenCamera#90]
The Android "accessibility icon" goes in the bottom right of portrait mode. (In open camera it's always in that physical corner).
With wide-enough photo dimensions chosen, the thumbnail ends up in the same place as the accessibility button.. and cannot be accessed.
![Screenshot_20200601-053124](https://user-images.githubusercontent.com/20318973/83409630-a0d09680-a3c9-11ea-93fe-3474455d6c6d.jpg)

**Comment by jaggzh** [comment]
Also in portrait mode. Still a problem in 2022. It's there a way to know, in Android, if that ui feature is enabled by the user?
![Screenshot_20220304-080811](https://user-images.githubusercontent.com/20318973/156798738-c4a82999-d87c-411d-a5c9-11d6fa97479b.jpg)

**Comment by jaggzh** [comment]
By the way, it means the photo can't be clicked to view the last or prior shots:
![Uploading Screenshot_20220304-080811.jpg…]()
